test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeControllerSpec.scala

```diff
@@ -869,7 +1149,8 @@ class RepresenteeControllerSpec
 
       val allQuestionsAnswers = IncompleteRepresenteeAnswers(
         Some(completeAnswers.name),
-        Some(completeAnswers.id)
+        Some(completeAnswers.id),
+        completeAnswers.dateOfDeath
```

Comment:
could we get a test that checks we redirect to the enter date of date page when it's supposed to happen?